doc/source/contributing_code.txt

```diff
@@ -145,8 +148,17 @@ hours).
 tests passing, and no warnings, the PR is merged into master by Jim
 (???)
```

Comment:
"merged to master by the integrator".  Doesn't have to be Jim unless he wants that role.